src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -165,15 +165,15 @@ def animal_energy_units(geom):
                      clipped_counties.*
               FROM clipped_counties
           )
-          SELECT SUM(beef_ha * totalha * clip_percent) AS beef_cows,
-                 SUM(broiler_ha * totalha * clip_percent) AS broilers,
-                 SUM(dairy_ha * totalha * clip_percent) AS dairy_cows,
-                 SUM(goat_ha * totalha * clip_percent) +
-                 SUM(sheep_ha * totalha * clip_percent) AS sheep,
-                 SUM(hog_ha * totalha * clip_percent) AS hogs,
-                 SUM(horse_ha * totalha * clip_percent) AS horses,
-                 SUM(layer_ha * totalha * clip_percent) AS layers,
-                 SUM(turkey_ha * totalha * clip_percent) AS turkeys
+          SELECT SUM(beef_ha * ag_ha * clip_percent) AS beef_cows,
+                 SUM(broiler_ha * ag_ha * clip_percent) AS broilers,
+                 SUM(dairy_ha * ag_ha * clip_percent) AS dairy_cows,
+                 SUM(goat_ha * ag_ha * clip_percent) +
```

Comment:
Was wondering whether this `+` here is a typo, but it looks like the app does not specify `goats` in GWLF-E settings -- https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/mmw/settings/gwlfe_settings.py#L467